modules/custom/az_marketing_cloud/templates/node--view--az-marketing-cloud--30-70-layout.html.twig

```diff
@@ -0,0 +1,128 @@
+ *     teaser listings.
+ *   - node--unpublished: Appears on unpublished nodes visible only to site
+ *     admins.
+ * - title_attributes: Same as attributes, except applied to the main title
+ *   tag that appears in the template.
+ * - content_attributes: Same as attributes, except applied to the main
+ *   content tag that appears in the template.
+ * - author_attributes: Same as attributes, except applied to the author of
+ *   the node tag that appears in the template.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ * - view_mode: View mode; for example, "teaser" or "full".
+ * - teaser: Flag for the teaser state. Will be true if view_mode is 'teaser'.
+ * - page: Flag for the full page state. Will be true if view_mode is 'full'.
+ * - readmore: Flag for more state. Will be true if the teaser content of the
+ *   node cannot hold the main body content.
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_node()
+ *
+ * @ingroup themeable
+ */
+#}
+
+<table cellpadding="0" cellspacing="0" width="100%" role="presentation"
+  style="text-align: left; min-width: 100%;"
+  class="stylingblock-content-wrapper">
+  <tbody>
+    <tr>
+      <td style="padding: 15px;" class="stylingblock-content-wrapper" align="left">
+
+        <table cellspacing="0" cellpadding="0" role="presentation" class="qs-layout-3070" style="width: 100%;">
+          <tbody>
+            <tr>
+              <td valign="top" class="responsive-td" style="width: 30%; padding-right: 7.5px;">
+                <table width="100%" cellspacing="0" cellpadding="0" role="presentation">
+                  <tbody>
+                    <tr>
+                      <td align="center">
+
+                        <!-- QS: IMAGE -->
+                        <img class="qs-img" src="{{ images.az_marketing_cloud_30_70_layout }}" alt="" width="200" style="display: block; height: auto; width: 100%;">
+
+                      </td>
+                    </tr>
+                  </tbody>
+                </table>
+              </td>
+              <td valign="top" class="responsive-td" style="width: 70%; padding-left: 7.5px;">
+                <table cellpadding="0" cellspacing="0" width="100%" role="presentation"
+                  style="text-align: left; min-width: 100%; " class="stylingblock-content-wrapper">
+                  <tbody>
+                    <tr>
+                      <td class="stylingblock-content-wrapper" align="left">
+
+                        <!-- QS: TITLE (recomended 80 character max) -->
```

Comment:
```suggestion
                        <!-- QS: TITLE (recommended 80 character max) -->
```

Reply:
Idea for the admin view… Possibly a validator that tells the user if a title/headline is too long…